.github/workflows/ci.yml

```
@@ -319,6 +319,7 @@ jobs:
               jq --arg version ${{ steps.version.outputs.version }} '
                 .version |= $version |
                 ."compiler-version" |= $version
+                optionalDependencies = (.optionalDependencies | .[] |= $version)
```

The syntax seems incorrect, I think it should be like this:

```suggestion
                ."compiler-version" |= $version |
                .optionalDependencies = (.optionalDependencies | .[] |= $version)
```

---

Also this will replace all values in the optionalDependencies map, which is file for [this package.json](https://github.com/sass/embedded-host-node/pull/158/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R28) but something we should be careful if for whatever reason we add a new optional dependency

Maybe worth adding a comment on the other repo about it?

---

I would add a comment... if `package.json` supported comments :/